ci-operator/step-registry/openstack/ovsdpdk/machineset/openstack-ovsdpdk-bindvfiodriver-ref.yaml

```diff
@@ -0,0 +1,15 @@
+ref:
+  as: openstack-ovsdpdk-bindvfiopcidriver
+  from: openstack-installer
+  commands: openstack-ovsdpdk-bindvfiopcidriver-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+    - name: VFIONETWORK
```

Comment:
you'll need OS_CLOUD

Reply:
We should not use any openstack commands here. We just need the network id. which we should already have.